src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs

```diff
@@ -11,56 +11,63 @@ open System.Runtime.CompilerServices
 open System.Runtime.Versioning
 
 // Package reference information
-type PackageReference = {
-    Include:string
-    Version:string
-    RestoreSources:string
-    Script:string }
-
+type PackageReference =
+    {
+      Include:string
+      Version:string
+      RestoreSources:string
+      Script:string
+    }
```

Comment:
It seems like the following style is most commonly used accross the compiler code.
```suggestion
type PackageReference =
    { Include:string
      Version:string
      RestoreSources:string
      Script:string }
```

Reply:
This style has been most common but doesn't work well when `///` comments and newline are added - see Typoe

We should move systematically to the second style and add `///` comments everywhere.  No more undocumented code.